app/survey_config/social_survey_config.py

```diff
@@ -29,8 +29,21 @@ def __post_init__(self):
                 }
             ]
 
+        upstream_base_url = f"{self.base_url}/{self.language_code}"
+        ons_url = ONS_URL_CY if self.language_code == "cy" else ONS_URL
+
         if not self.account_service_log_out_url:
-            self.account_service_log_out_url: str = f"{self.base_url}/sign-in/logout"
+            self.account_service_log_out_url: str = f"{upstream_base_url}/start/"
+
+        self.account_service_log_out_url: str = f"{upstream_base_url}/start"
```

Comment:
Why is this set again?
```suggestion
```

Reply:
Removed now, some kind of human error In left it there, thanks!